policy/src/type/permit-application.type.ts

```diff
@@ -0,0 +1,62 @@
+type PermitMailingAddress = {
```

Comment:
looks like some interfaces that we could put into their own repo and pull from across the project -- `vehicles`, `dops` and `frontend` (and here) when needed. #futuretask

Reply:
100%. A central place to define the structure of the permit application would be fantastic.